tests/test_utils.py

```diff
@@ -174,3 +179,44 @@ def test_collect_args_local(wps_test_collect_args, file1, file2, argc):
 )
 def test_collect_args_online(wps_test_collect_args, file1, file2, argc):
     collect_args_test(wps_test_collect_args, file1, file2, argc)
+
+
+@pytest.mark.parametrize(
+    ("package"), [("base"), ("utils")],
+)
+def test_get_package(package):
+    pkg = get_package(package)
+    assert pkg.__dict__["__rname__"] == package
+
+
+@pytest.mark.parametrize("package", ["invalid_pkg"])
+def test_get_package_err(package):
+    with pytest.raises(ProcessError) as e:
+        get_package(package)
+        assert str(vars(e)["_excinfo"][1]) == f"R package, {package}, is not installed"
+
+
+@pytest.mark.parametrize(
+    ("r_file", "r_object_name"),
+    [(resource_filename(__name__, "data/expected_gsl.rda"), "expected_gsl_vector")],
+)
+def test_load_rdata_to_python(r_file, r_object_name):
+    r2py_object = load_rdata_to_python(r_file, r_object_name)
+    assert "robjects" in str(type(r2py_object))
```

Comment:
What happens if an invalid `r_object_name` is given? Does `load_rdata_to_python` even return something? Or None type? If the function terminates due to wrong input, does the `assert` statement do anything?

Reply:
@sum1lim Ya I wasn't sure about this test. Do you have any suggestions for what to test for?

Reply:
I think the first thing to check is what happens if invalid input is provided. If the function returns None type, I think the test is good to go, but if an error is produced, I think `try except` block should be used to raise `ProcessError`. This applies to `save_python_to_rdata` as well.

Reply:
Why does it have to be a Processes error? what's wrong with the existing error? just wondering.

Reply:
@cairosanders Ummm, I am not sure. But I guess it is because the error is caught outside of the function/method that is being wrapped, so using ProcessError defined by PyWPS with an appropriate message makes more sense? I actually have never thought about this topic. Maybe leaving the functions the way it is could be OK. The point I wanted to address here is the use of the assert statement. Using [pytest.raises](https://docs.pytest.org/en/reorganize-docs/new-docs/user/pytest_raises.html) might be more appropriate here.